samtranslator/swagger/swagger.py

```diff
@@ -285,6 +286,108 @@ def _make_cors_allowed_methods_for_path(self, path):
         # Allow-Methods is comma separated string
         return ','.join(allow_methods)
 
+    def add_authorizers(self, authorizers):
+        """
+        Add Authorizer definitions to the securityDefinitions part of Swagger.
+
+        :param list authorizers: List of Authorizer configurations which get translated to securityDefinitions.
+        """
+        self.security_definitions = self.security_definitions or {}
+
+        for authorizerName, authorizer in authorizers.items():
+            self.security_definitions[authorizerName] = authorizer.generate_swagger()
+
+    def set_path_default_authorizer(self, path, default_authorizer, authorizers):
+        """
+        Sets the DefaultAuthorizer for each method on this path. The DefaultAuthorizer won't be set if an Authorizer
+        was defined at the Function/Path/Method level
+
+        :param string path: Path name
+        :param string default_authorizer: Name of the authorizer to use as the default. Must be a key in the authorizers param.
+        :param list authorizers: List of Authorizer configurations defined on the related Api.
+        """
+        for method_name, method in self.paths[path].items():
+            self.set_method_authorizer(path, method_name, default_authorizer, authorizers, default_authorizer=default_authorizer, is_default=True)
+
+    def add_auth_to_method(self, path, method_name, auth, api):
+        """
+        Adds auth settings for this path/method. Auth settings currently consist solely of Authorizers
+        but this method will eventually include setting other auth settings such as API Key,
+        Resource Policy, etc.
+
+        :param string path: Path name
+        :param string method_name: Method name
+        :param dict auth: Auth configuration such as Authorizers, ApiKey, ResourcePolicy (only Authorizers supported
+                          currently)
+        :param dict api: Reference to the related Api's properties as defined in the template.
+        """
+        method_authorizer = auth and auth.get('Authorizer')
+        if method_authorizer:
```

Comment:
It also seems like tests may not be hitting this section of code. Can you verify?

Reply:
(Test readout says that [325->exit] and [360->exit] are not being tested)

Reply:
The terminal coverage report isn't clear. If you look at the html report it is a "partial" miss. In this case it's saying that `method_authorizer` was never false.